esp/src/eclwatch/FilterDropDownWidget.js

```diff
@@ -151,13 +151,15 @@ define([
 
         refreshState: function () {
             if (this.exists()) {
-                this.iconFilter.src = Utility.getImageURL("filter1.png");
+                this.iconFilter.style.color = "cornflowerblue";
```

Comment:
Where did `cornflowerblue` come from? Does it match our currently color?

Reply:
It matched better than the blue color, and it matched our current color.

Reply:
It matched better than the blue color, and it matched our current color.

Reply:
@miguelvazq you can open a ticket to start relocating these into the new theme file so that they can get referenced by "concept" rather than specific color?

Reply:
> It matched better than the blue color, and it matched our current color.

Couldn't you match exactly using ColorZilla eye dropper tool to make sure we are consistent?

Reply:
> @miguelvazq you can open a ticket to start relocating these into the new theme file so that they can get referenced by "concept" rather than specific color?

@GordonSmith @kunalaswani I will change these in our material-ui theme. And give them our palette treatment in there since the theme can support FA as well.